pandas/core/indexes/datetimes.py

```diff
@@ -1735,7 +1735,33 @@ def freq(self, value):
     is_quarter_start = _field_accessor(
         'is_quarter_start',
         'is_quarter_start',
-        "Logical indicating if first day of quarter (defined by frequency)")
+        """
+        Return a boolean indicating whether the date is the first day of a
+        quarter.
+
+        Returns
+        -------
+        is_quarter_start : Series of boolean.
+
+        See Also
+        --------
+        quarter : Return the quarter of the date.
```

Comment:
See currently unresolved discussion in https://github.com/pandas-dev/pandas/pull/20275#discussion_r173651295